client/utils/mockData.ts

```diff
@@ -1,6 +1,14 @@
 import { faker } from '@faker-js/faker'
 
-export const generateMockUserData = (count: number) => {
+interface UserData {
+    id: string
+    name: string
+    email: string
+    sex: string
+    // Add more fields as needed
```

Comment:
No need for this comment here. Also the sex field is not something returned from the backend currently, maybe it could be removed so your mock data is more alike actual production data?